src/pages/library/DataBrowserPreview.js

```diff
@@ -0,0 +1,109 @@
+import _ from 'lodash/fp'
+import { useState } from 'react'
+import { div, h, h1 } from 'react-hyperscript-helpers'
+import { Select } from 'src/components/common'
+import FooterWrapper from 'src/components/FooterWrapper'
+import { centeredSpinner, spinner } from 'src/components/icons'
+import { libraryTopMatter } from 'src/components/library-common'
+import { SimpleTable } from 'src/components/table'
+import { Ajax } from 'src/libs/ajax'
+import colors from 'src/libs/colors'
+import { withErrorReporting } from 'src/libs/error'
+import * as Utils from 'src/libs/utils'
+import { useDataCatalog } from 'src/pages/library/dataBrowser-utils'
+
+
+const styles = {
+  table: {
+    header: {
+      color: colors.accent(),
+      height: '1rem',
+      textTransform: 'uppercase', fontWeight: 600, fontSize: '0.75rem'
+    }
+  }
+}
+
+const activeTab = 'browse & explore'
+let metadata
+let tableMap
```

Comment:
These deserve explanation, either in PR comments or code comments.

Reply:
Made these updates!